python/coral_usb/util.py

```diff
@@ -30,6 +30,33 @@ def get_panorama_sliced_image(panorama_img, panorama_slice):
     return img
 
 
+def get_tile_slices(img_height, img_width, overlap=True,
```

Comment:
```suggestion
def get_tile_slices(height, width, overlap=True,
```

sorry, but `height` and `width` might be better.